### DISCUSSION TITLE
Adding new Turkmen localization

### DISCUSSION THREAD

**Discussion by PavelShevchenko1** [home-assistant/frontend#16580] [Localization]
English Name: Turkmen
Native Name: Türkmen

**Comment by PavelShevchenko1** [comment]
We had the task of implementing access to smart home control with Home Assistant to the local population. But I could not find the localization of the project in Turkmen.

**Reply by spacegaier** [reply]
The localization is handled by volunteers from the community. Are you saying you would be interested in starting the translation efforts to Turkmen?
